ci/README.md

```diff
@@ -107,6 +153,101 @@ The subsequent PRs on community will be able to pass the
 `check-merge-to-enterprise` check as long as they don't have a conflict with
 `enterprise-master`.
 
+### What to do when your branch got outdated?
+
+So there's one issue that can occur. Your branch will become outdated with
+master and you have to make it up to date. There are two ways to do this using
+`git merge` or `git rebase`. As usual, `git merge` is a bit easier than `git
+rebase`, but clutters git history. This section will explain both. If you don't
+know which one makes the most sense, start with `git rebase`. It's possible that
+for whatever reason this doesn't work or becomes very complex, for instance when
+new merge conflicts appear. Feel free to fall back to `git merge` in that case,
+by using `git rebase --abort`.
+
+#### Updating both branches with `git rebase`
+
+In the community repo, first update the outdated branch using `rebase`:
+
+```bash
+git checkout $PR_BRANCH
+# Keep a backup in case you want to fallback to the merge approach
+git checkout -b ${PR_BRANCH}-backup
+git checkout $PR_BRANCH
+# Actually update the branch
+git fetch origin
+git rebase origin/master
+git push origin $PR_BRANCH --force-with-lease
+```
+
+In the enterprise repo, rebase onto the new community branch with
+`--preserve-merges`:
+
+```bash
+git checkout $PR_BRANCH
```

Comment:
it might be good to create a backup branch here as well.

Reply:
I didn't put it there, since here you would normally revert to the original by doing `git rebase --abort`. If `rebase` succeeded you are done. In the rare case where after you are done you find out that something is bad, there's always `git reset origin/$PR_BRANCH` or `git reflog`.